src/components/VideoList/VideoListCard/VideoListCard.component.jsx

```diff
@@ -1,74 +1,94 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 
 import { useHistory } from 'react-router-dom';
 
 import he from 'he';
 
-import { queryWatchUrl } from '../../../utils/functions/routes';
+import { queryWatchUrl, queryPrivateWatchUrl } from '../../../utils/functions/routes';
+import WatchLaterButton from '../../WatchLater/WatchLaterIconButton';
 
-import {
-  VideoCardContainer,
-  VideoThumbnails,
-  InfoContainer,
-  ThumbnailContainer,
-  Title,
-  ExtraInfoDiv,
-  Author,
-  Overlay,
-} from './VideoListCard.styled';
+import * as Styled from './VideoListCard.styled';
 
-const VideoListCard = ({ title, author, description, videoId, thumbUrl }) => {
+const VideoListCard = ({
+  title,
+  channelTitle,
+  description,
+  videoId,
+  thumbUrl,
+  isInPrivateRoute,
+  videoInfo,
+  removeThisItem,
+}) => {
   const history = useHistory();
+  const [isWatchLaterSelected, setIsWatchLaterSelected] = useState(true);
 
   const goToVideo = (e) => {
     e.preventDefault();
-    const watchURL = queryWatchUrl(videoId);
-    history.push(watchURL);
+    if (isInPrivateRoute) {
+      // can safely do this because only selected are shown
+      history.push(queryPrivateWatchUrl(videoId));
+    } else {
+      history.push(queryWatchUrl(videoId));
+    }
   };
 
   const titleFull = he.decode(title);
-  const authorFull = he.decode(author);
+  const channelTitleFull = he.decode(channelTitle);
   const descriptionFull = he.decode(description);
 
   const shortTitle = titleFull.length > 55 ? `${titleFull.slice(0, 52)}...` : titleFull;
-  const shortAuthor =
-    authorFull.length > 55 ? `${authorFull.slice(0, 52)}...` : authorFull;
+  const shortchannelTitle =
```

Comment:
Truncating a string is looking more recurrent here. Perhaps consider adding a util that receives a string and a desired length, and it returns the truncated versions if larger, or the original if shorter.